src/stylesheets/ff-components.scss

```diff
@@ -383,6 +383,9 @@ li.ff-list-item {
   z-index: 2;
   width: 100%;
   max-width: 42rem;
+  max-height: 100%;
```

Comment:
do we want to let it get this large, should we reduce to say `calc(100% - 120px)` or something like that?

Reply:
Our header is a static 60px - so I'd vote we go for `60 * 2 + (2 * 12) = 144px` padding. - prevents overlap with the header, adds a sensible (12px) margin and ensures symmetry top/bottom

Reply:
I think that's possibly a bit too much. The following looks odd for the dialog to not have been just a little bit bigger. Maybe I just got unlucky with the size of my window and this particular dialog.

<img width="685" alt="image" src="https://user-images.githubusercontent.com/51083/175569357-3d5a342e-5f7f-4800-9c4b-e6e59ab44f62.png">

But it does help solve #21 

I suspect the right longer-term answer is to avoid dialog boxes for long forms - only for quick actions or confirmations. For 'creating' or 'editing' type actions, we could standardise on using dedicated pages for them - like we do the Create Project page.

Reply:
Yeah, I feel like standardising into pages makes more sense - we're only going to be increasing option counts... I also don't like how the actions are included in the scroll, the content should scroll, but not the `actions` slot.

Reply:
>  don't like how the actions are included in the scroll

Agreed - will fix up.

Reply:
Just pushed a slight variation

 - max height set to `calc(100% - 84px)` - that's `60px` less than your suggestion
 - margin-top set to `60px` - to ensure it sits below the title bar
 - the net result is the dialog is centred vertically in the 'body' of the page
 - Inside the dialog, body scrolls, actions stay at the bottom. Visually, there's no obvious border between the actions and the body - so the content just disappears. I did try adding a border, but that looks wrong for the simple dialogs that don't require scrolling.... but still, getting better.

![ejqAEcZ30B](https://user-images.githubusercontent.com/51083/175665478-2f962777-48ca-47a5-86e2-0df96e270dad.gif)

Reply:
Much better, the only thing I could think of with the styling is to add the border or box-shadow when the content requires scrolling. -will require some JS trickery - but happy to approve this for now.